content/blog/tikv-3.0ga-release-notes.md

```diff
@@ -0,0 +1,160 @@
+
+In this version, we tackled many problems of stability at massive scales. Whether it's spanning hundreds of nodes or storing over a trillion key-value pairs, we've seen our users and contributors put TiKV to the test in serious, real world, production scenarios. With 3.0, we've taken our ideas and lessons to bring a host of features that can better support these growing demands.
+
+## Steady at scale
+
+In TiKV 3.0 we've improved our system by:
+
+* **Optimizing the Raft heartbeat mechanism.** With the Hibernate Region feature, TiKV now adjusts the heartbeat frequency according to region activity. This means you'll see less CPU time and network traffic from idle regions.
+
+* **Distributing Garbage Collection.** The introduction of a distributed garbage collector improves performance on large scale clusters dramatically, leading to better stability through more consistent performance.
+
+* **Pessimistic Locking.** It's now possible to ask TiKV to treat your transactions pessimistically. This means you can take exclusive ownership over a value for a duration. As Rust developers, we really like the idea of ownership!
+
+* **Expanding our coprocessor.** With lots of new or improved functionalities such as vector operations, batch executor, RPN functions, `work-stealing` thread pool model,  our coprocessor continues to evolve, allowing for more efficient ways to work with your data.
+
+* **Enhancing operator friendliness.** Human or machine, we've empowered our operators to get more out of TiKV by unifying our log format, adding new features to `tikv-ctl`, adding even more in-depth metrics, and serving HTTP based metrics. This makes TiKV easier to operate, inspect, and enjoy.
+
+* **Refining request types.** While TiKV previously supported commands like `BatchGet`, 3.0 brings a new `BatchCommands` request type. This allows TiKV to handle batches of requests of differing kinds (such as `Get` and `Put`), leading to less data on the wire and better performance. We also added support for raw reversed scanning, and `Insert` semantics on prewrite.
+
+* **Reducing write amplification.** Inspired by the great ideas from [WiscKey](https://www.usenix.org/system/files/conference/fast16/fast16-papers-lu.pdf), we implemented a storage engine plugin to RocksDB that we dubbed 'Titan'. It works best with larger (>1 KB) values.
+
+You can see all the changes in detail [here](https://github.com/tikv/tikv/blob/release-3.0/CHANGELOG.md).
+
+## Improved Performance
+
+Using [`go-ycsb`](https://github.com/pingcap/go-ycsb) we benchmarked TiKV 3.0.0 against TiKV 2.1.14. We benchmarked a cluster of 3 TiKV nodes, 1 PD node, and 1 node running YCSB. We used DigitalOcean `s-8vcpu-32gb` size machines, and you can reproduce the benchmark for yourself using the fully automated terraform script [here](https://github.com/Hoverbear/tikv-bench).
```

Comment:
We should move the script to tikv org before publishing this

Reply:
I don't feel comfortable sanctioning it as an official benchmarking script right now. Having the repo under my name right now is a signal that it's not meant to be taken as a production script.

Reply:
What do you think @dcalvin ?

Reply:
Updated to https://github.com/tikv/terraform-tikv-bench

Reply:
Good move.